test/marketplace/ZcToken.sol

```diff
@@ -18,7 +18,7 @@ contract ZcToken {
   string public symbol;
   uint8 public decimals;
   
-  mapping (address => uint256) private balances;
+  mapping (address => uint256) private balanceOfReturn;
```

Comment:
so this is a little diff that the simpler way its in the Erc20 mock. That one does have the limitaton of only being able to have 1 set a time to return, so if we want multiple different balances to return this is fine if that is needed.